README.md

```diff
@@ -43,17 +43,13 @@ Commonly referenced packages:
 [metapackage-myget-badge]: http://img.shields.io/dotnet.myget/aspnetcore-dev/v/Microsoft.AspNetCore.svg?style=flat-square&label=myget
 
 Package                           | MyGet
-----------------------------------|-------------------
+:---------------------------------|:---------------------------------------------------------
 Microsoft.AspNetCore.All          | [![][all-metapackage-myget-badge]][all-metapackage-myget]
 Microsoft.AspNetCore              | [![][metapackage-myget-badge]][metapackage-myget]
 
 ### ASP.NET Core Shared Framework
 
-[win-x64-badge]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-win-x64-version-badge.svg
-[win-x86-badge]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-win-x86-version-badge.svg
-[linux-x64-badge]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-linux-x64-version-badge.svg
-[osx-x64-badge]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-osx-x64-version-badge.svg
-
+[badge-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/aspnetcore-runtime-win-x64-version-badge.svg
```

Comment:
Why only the win-x64 badge?

Reply:
I can't imagine a scenario in which we will actually have different versions per platform. So, instead of a version badge for each cell in the table, I added only one patch per column. win-x64 happened to be the first in the list of badges.

Reply:
Okay